redbeat/schedulers.py

```diff
@@ -119,9 +119,13 @@ def get_redis(app=None):
     app = app_or_default(app)
     conf = ensure_conf(app)
     if not hasattr(app, 'redbeat_redis') or app.redbeat_redis is None:
-        redis_options = conf.app.conf.get(
-            'REDBEAT_REDIS_OPTIONS', conf.app.conf.get('BROKER_TRANSPORT_OPTIONS', {})
+        redis_options = conf.app.conf.first(
+            "redbeat_redis_options", "REDBEAT_REDIS_OPTIONS"
         )
+        if redis_options is None:
+            redis_options = conf.app.conf.first(
+                "BROKER_TRANSPORT_OPTIONS", "broker_transport_options"
```

Comment:
Is there a reason not to consistently prefer lowercase option names over UPPERCASE ones?

Reply:
Celery [said](https://docs.celeryproject.org/en/stable/userguide/configuration.html#new-lowercase-settings) they will support uppercase until v6.0.

When supplying uppercase only config, and reading lowercase first, it returns an empty dict because it's reading the default lowercase set by Celery.

Reply:
cool, I assumed due to line 123, https://github.com/sibson/redbeat/pull/182/files#diff-3306d7d435bbd01ef84d83cae122c8cfR123, that lower case first was preferred.  Should line 123 be updated to also be uppercase first?

Reply:
I kept that lower first to match the other config code (https://github.com/sibson/redbeat/blob/350c02a6281a3ba48b896ba7d60ea7c7c5d17512/redbeat/schedulers.py#L193)

The test cases pass for either lower first or upper first cases for redbeat_redis_options.

Reply:
got it thanks for explaining.  I wonder if the single UPPERCASE first is too clever to be maintainable.  Is there a more obvious way we could get the same behaviour? If not, that's ok, I'll merge as is.

Reply:
Hmmm. I don't think there is a clever way.

Celery supports both, so I think we'd have to check both too.

A user could use all uppercase (even though it's deprecated) in celery 5.0 without errors.